src/python/pants/invalidation/build_invalidator.py

```diff
@@ -108,6 +138,15 @@ def create(cls, build_task=None):
       root = os.path.join(cls.global_instance().get_options().pants_workdir, 'build_invalidator')
       return BuildInvalidator(root, scope=build_task)
 
+  @staticmethod
```

Comment:
There are quite a few definitions of `def cacheable`, but this one in particular doesn't seem to be carrying its weight relative to a direct call... since it's static, I don't expect that it's here for the purposes of mocking...?

Reply:
The driver was isolating knowledge of CacheKey.cacheable to `src/python/pants/invalidation/build_invalidator.py`. All the rest of the forwarders sprouted from that single concern. Basically wading through this code pulls up alot of interdeps and via alot of `x.y.z.w` accesses indicative of suboptimal factoring. I could unwind these to raw `vts.cache_key.cacheable` up in Task, not super thrilled at that prospect though.